GitUI/GitUIExtensions.cs

```diff
@@ -71,7 +71,7 @@ public static class GitUIExtensions
                     ? $"{firstId}...{item.SecondRevision.ObjectId}"
                     : $"{item.BaseA}..{firstId} {item.BaseB}..{item.SecondRevision.ObjectId}";
 
-                await fileViewer.ViewTextAsync("range-diff.sh", $"git range-diff {range}");
+                await fileViewer.ViewTextAsync("git-range-diff.sh", $"git range-diff {range}");
```

Comment:
This is just a dummy file name, isn't it?

Reply:
Yes, for syntax highlighting